guides/17_Migration_to_3_0.md

```diff
@@ -123,8 +132,60 @@ const eventReporter = new EventReporter(...)
+| 3   | observableMetricSpec      | googNackCountReceivedPerSecond | Removed | Use __nackCount__ instead                  |                                                                                                                                                                                                           |
+| 4   | observableMetricSpec      | availableSendBandwidth         | Removed | Use __availableOutgoingBitrate__ instead   |                                                                                                                                                                                                           |
+| 5   | observableMetricSpec      | availableReceiveBandwidth      | Removed | Use __availableIncomingBitrate__ instead   |                                                                                                                                                                                                           |
+| 6   | observableVideoMetricSpec | videoUpstreamGoogFrameHeight   | Removed | Use __videoUpstreamFrameHeight__ instead   |                                                                                                                                                                                                           |
+| 7   | observableVideoMetricSpec | videoUpstreamGoogFrameWidth    | Removed | Use __videoUpstreamFrameWidth__ instead.   |                                                                                                                                                                                                           |
+| 8   | observableVideoMetricSpec | videoDownstreamGoogFrameHeight | Removed | Use __videoDownstreamFrameHeight__ instead |                                                                                                                                                                                                           |
+| 9   | observableVideoMetricSpec | videoDownstreamGoogFrameWidth  | Removed | Use __videoDownstreamFrameWidth__ instead  |                                                                                                                                                                                                           |
+
+#### Get raw RTCStatsReport
+
+We add a new `rtcStatsReport` property to `DefaultClientMetricReport` to store raw [`RTCStatsReport`](https://developer.mozilla.org/en-US/docs/Web/API/RTCStatsReport) and expose it via `metricsDidReceive(clientMetricReport: ClientMetricReport)` event. You can get the `rtcStatsReport` via `clientMetricReport.getRTCStatsReport()`. These metrics are updated every second.
```

Comment:
Can we mention that this should be used instead of the one in `audioVideoController` to avoid excessive call to `getStats` API and the other one might be deprecated in the future.